common/changes/office-ui-fabric-react/feedbacksection_2018-07-11-20-48.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Added optional feedback section for component pages",
+      "type": "minor"
```

Comment:
DocPage.types.ts is not part of the API surface I dont' think (not exported from the root folder) so this can be a patch.. probably.